src/tracking-buffer/bigint.js

```diff
@@ -66,15 +68,27 @@ function convertLEBytesToString(buffer) {
 module.exports.numberToInt64LE = numberToInt64LE;
 function numberToInt64LE(num) {
   // adapted from https://github.com/broofa/node-int64
-  const negate = num < 0;
-  let hi = Math.abs(num);
-  let lo = hi % 0x100000000;
-  hi = (hi / 0x100000000) | 0;
+  let hi, lo;
```

Comment:
This would only help for the case where the large integer value is specified as a string. For the cases, where the value is specified as a number, like below, the value is corrupted before execution gets here.
```JavaScript
commandBigInt.addParameter('Id', TYPES.BigInt, 9007199254740999);
```

https://github.com/tediousjs/tedious/pull/504 handles this by adding range validation with an error messages that suggests specifying the parameter as VarChar.

We have two options:

- Take this change and tweak the validation in https://github.com/tediousjs/tedious/pull/504 to only enforce when the specified value is a number. Any existing code using BigInt with large numbers will magically start working correctly.
- Discard this PR and just stick with blanket validation in https://github.com/tediousjs/tedious/pull/504 . This keeps the code and the story simple and alerts any existing code using large numbers with BigInt that there was data corruption.

I'm leaning towards the latter option.